opencti-platform/opencti-front/src/private/components/observations/indicators/DecayDialog.tsx

```diff
@@ -87,13 +87,31 @@ const DecayDialogContent : FunctionComponent<DecayDialogContentProps> = ({ indic
     };
   };
 
+  const getDisplayForUpcomingUpdates = (history: DecayHistory) => {
+    if (history.score === indicator.decay_applied_rule?.decay_revoke_score) {
+      return {
+        label: t_i18n('Revoke score'),
+        style: { color: theme.palette.secondary.main },
+        score: history.score,
+        updated_at: getDateAsTextFor(history),
+      };
+    }
+    return {
+      label: t_i18n('Stability threshold'),
+      style: { color: theme.palette.text.primary },
+      score: history.score,
+      updated_at: getDateAsTextFor(history),
+    };
+  };
+
   const labelledHistoryList: LabelledDecayHistory[] = [];
-  decayHistory.forEach((history) => (
-    labelledHistoryList.push(getDisplayFor(history))
+  const currentScoreIndex = decayHistory.findLastIndex((history) => history.score === indicator.x_opencti_score);
```

Comment:
fix display of current score when we have multiple times in history the same score (it was displayed "Current stable score" each time)